packages/base/src/util/parseProperties.js

```diff
@@ -15,6 +15,12 @@ const mEscapes = {
 	"\\t": "\t",
 };
 
+/**
```

Comment:
We should proxy this to `AssetRegistry.js` or some other public file. The `util/` directory is private and subject to change.

Reply:
This should be a separate import, a proxy import will always include this (if no tree shaking is present).
How about a separate module `@ui5/webcomponents-base/dist/PropertiesParser.js`?

Reply:
`@ui5/webcomponents-base/dist/PropertiesFileFormat.js`